index.js

```diff
@@ -104,6 +104,13 @@ class Matrix {
         this.colVisit = true;
         return this.colMemo = this.transpose().rowVectors;
     }
+    get diagonalVector() {
```

Comment:
I just wanted it to be uniform with the other getters (rowVectors and colVectors)